chapter08/red/exercise2.pl

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+my %handles;
+while (<>) {
+  my ( $castaway ) = split /:/;
+
+  if ( !defined $handles{$castaway} ) {
+    open $handles{$castaway}, '>', lc $castaway . '.log'
```

Comment:
Good point. I'm alright with it just dying on a name it can't write - there could be any number of OS-specific rules that `open` will fail on.